src/types/commands.ts

```diff
@@ -119,6 +120,25 @@ export const invalidateEvaluationCommands = new Set<CommandTypes>([
   "UPDATE_LOCALE",
 ]);
 
+const invalidateTableStyleCommands = [
+  "HIDE_COLUMNS_ROWS",
+  "UNHIDE_COLUMNS_ROWS",
+  "UNFOLD_HEADER_GROUP",
+  "FOLD_HEADER_GROUP",
+  "FOLD_ALL_HEADER_GROUPS",
+  "UNFOLD_ALL_HEADER_GROUPS",
+  "CREATE_TABLE",
+  "UPDATE_TABLE",
+  "UPDATE_FILTER",
+] as const;
+const invalidateTableStyleCommandsSet = new Set<CommandTypes>(invalidateTableStyleCommands);
+
+export function doesCommandInvalidatesTableStyle(
```

Comment:
I'd move this out of the `types` folder. There's isn't even a type associated with this function